### DISCUSSION TITLE
Switching Phones Mid-Sensor

### DISCUSSION THREAD

**Discussion by noctividus** [NightscoutFoundation/xDrip#2479] [Q&A]
I'm getting a new phone and I'm in the middle of a sensor. I understand how to move the database over, but I'm up on NightScout, so that's not as important as not losing 8 days on a sensor. Are there instructions somewhere about disabling the old phone as sync master and setting up the new phone as the new collector mid-sensor? I'm using a Dexcom G6.

**Comment by Navid200** [comment]
Whatever you do, do NOT tap on stop sensor.

After restoring the database on the new phone, make sure you can see history.  If not, restore three times.  Do you see history?
Resolve this before moving ahead.  

On the new phone, set everything up to be the new collector except entering the transmitter ID (https://navid200.github.io/xDrip/docs/G6-Recommended-Settings).

On the old phone, disable collection (https://navid200.github.io/xDrip/docs/Stop-xDrip).

Now, on the new phone, enter the transmitter ID.  xDrip may ask you to either change settings or start sensor.  Do neither!
Follow these steps if needed to get readings:  https://navid200.github.io/xDrip/docs/Connectivity-troubleshoot

**Reply by steve8x8** [reply]
Interestingly, David Burren (BionicWookiee) suggests a different procedure in https://bionicwookiee.com/2022/10/27/moving-a-dexcom-g6-sensor-between-phones/ - which involves stopping the sensor, but not getting the command sent to the transmitter.
He doesn't mention neither xDrip nor the native G6 app, but it seems that it would apply to both; as long as the transmitter is asleep the (admittedly dangerous) command won't reach it, it just establishes a clean state of the app. Of course this state should not be transferred to the other phone anymore.
If the xDrip DB doesn't have to be kept, that would clearly require a (re)connect to the sensor (aka start), wouldn't it?

**Reply by Navid200** [reply]
No,

Disagree.
There is absolutely no need to start sensor if all you want to do is to establish connectivity with a session in progress.
I don't create a page in my guide that I have not tested already:
https://navid200.github.io/xDrip/docs/Session-in-progress

**Reply by steve8x8** [reply]
OK, thanks
